code/modules/uplink/uplink_items.dm

```diff
@@ -1590,8 +1566,6 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	purchasable_from = UPLINK_NUKE_OPS | UPLINK_CLOWN_OPS
 	restricted = TRUE
 
-//SKYRAT EDIT REMOVAL BEGIN
-/*
 /datum/uplink_item/device_tools/suspiciousphone
```

Comment:
The CRAB protocol hasn't resulted in any roleplay from the antagonist that used it in almost every single instances I've seen it used. They hide in maint, turn it on and then profit.

Reply:
I'm leaving the CRAB available because it's an extremely effective "get that guy who's sat in the same spot for 5 hours to leave their chair for a few minutes" tool, allowing for antagonists to do things while folks are distracted.